CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   - AWSEC2
   - AWSTranscribe
 
+- Fix for decoding AWSPinpointEndpointProfile when it includes attributes ([PR #3601](https://github.com/aws-amplify/aws-sdk-ios/pull/3601))
```

Comment:
Don't forget `bug fixes` header, and move it above "Misc Updates"
```markdown
### Bug fixes

- Fix for decoding AWSPinpointEndpointProfile when it includes attributes ([PR #3601](https://github.com/aws-amplify/aws-sdk-ios/pull/3601))
```

Reply:
I've made the update and pushed to this branch.